### PR TITLE
Detect api style based on mimetype or apihub-style label

### PR DESCRIPTION
**Pull request by giteshk**
Added the ability to detect renderer based on the apihub-style label for hosted Registry service.
Added documentation to allow for running the spec renderer on GCE

### CODE REVIEW

File: containers/registry-spec-renderer/index.js [hunk 1]
```diff
@@ -42,57 +43,87 @@ app.use('/renderer/graphql', express.static('node_modules/react'))
...
+  switch (apiFormat) {
+    case "openapi":
+      renderer_template = swagger_ui_template.toString();
+      break;
+    case "asyncapi":
+      renderer_template = async_template.toString();
+      break;
+    case "graphql":
+      renderer_template = graphiql_template.toString();
+      break;
```
[the remark this answers is not in the document]
Comment (giteshk) [reply containers/registry-spec-renderer/index.js:57]: The default behaviour is to return the spec if the format is not supported.

File: containers/registry-spec-renderer/index.js [hunk 2]
```diff
@@ -42,57 +43,87 @@ app.use('/renderer/graphql', express.static('node_modules/react'))
...
+    res.setHeader("content-type", "text/html; charset=UTF-8");
+    hbstemplate = handlebars.compile(renderer_template);
+    res.send(hbstemplate({specUrl: specUrl}));
+  } else {
+    res.redirect(specUrl);
+  }
+  res.end();
+}
+
+function discoverAPIFormat(text) {
```
[the remark this answers is not in the document]
Comment (giteshk) [reply containers/registry-spec-renderer/index.js]: addressed this change

File: containers/registry-spec-renderer/index.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (shrutiparabgoogle) [reply containers/registry-spec-renderer/index.js:57]: Yes, I understand, to make it clear what I am suggesting is returning the spec in the `switch` statement's `default` case.

[the remark this answers is not in the document]
Comment (giteshk) [reply containers/registry-spec-renderer/index.js:78]: @shrutiparabgoogle I made it more obvious on what we are doing. thanks for the suggestion.

**Review by timburks — APPROVED** [review]

**Pull request merged**